src/main/java/dev/slimevr/bridge/ProtobufBridge.java

```diff
@@ -131,7 +132,8 @@ protected void positionReceived(Position positionMessage) {
 		if (tracker != null) {
 			if (positionMessage.hasX())
 				tracker.position.set(positionMessage.getX(), positionMessage.getY(), positionMessage.getZ());
-			tracker.rotation.set(positionMessage.getQx(), positionMessage.getQy(), positionMessage.getQz(), positionMessage.getQw());
+			tracker.rotation.set(positionMessage.getQx(), positionMessage.getQy(), positionMessage.getQz(),
```

Comment:
We should decide whether we want the behavior to be "fill to max col limit then wrap to newline", or "if there is a need to wrap, break every argument into its own newline".

I prefer the latter, as it is what `black` and `cargo fmt` use and I find it looks elegant, and reduces diff noise. The current formatting uses the former. What are y'alls thoughts?

Reply:
That's not possible in Eclipse formatter, I discussed it in discord. It can be formatted by hand like this or a post-processor can do this.

Reply:
Its actually possible, it depends on the wrapping mode. You can see an example here, in this case one of the new() calls is not wrapped because it doesn't hit the 100 character limit, and the other is wrapped, with each argument on a new line:

![image](https://user-images.githubusercontent.com/6969415/167351954-6d630228-24a4-46d4-ad20-fabc09c4c9bd.png)

You can control this behavior in the GUI by clicking on this:
![image](https://user-images.githubusercontent.com/6969415/167352060-9e37e958-e62c-4e9f-aef3-608e33776a4a.png)

Reply:
That's what I did and it wrapped EVERYTHING
![img](https://cdn.discordapp.com/attachments/823658497454374962/972866614850621450/unknown.png)

Reply:
Please check my configuration, it should handle this better